selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/webdriver/SeleniumWebDriverHelper.java

```diff
@@ -674,6 +674,37 @@ public void waitTextPresence(By elementLocator, String expectedText) {
     waitTextPresence(elementLocator, expectedText, DEFAULT_TIMEOUT);
   }
 
+  /**
+   * Waits during {@code timeout} until specified {@code element} does not contain the specified
+   * {@code expectedText}.
+   *
+   * <p>Note! Text is extracted by {@link WebElement#getText()} method.
+   *
+   * @param element element which should be checked
+   * @param expectedText text which should be not presented
+   * @param timeout waiting time in seconds
+   */
+  public void waitTextIsNotPresence(WebElement element, String expectedText, int timeout) {
+    webDriverWaitFactory
+        .get(timeout)
+        .until(
+            (ExpectedCondition<Boolean>)
+                driver -> !(waitVisibility(element).getText().contains(expectedText)));
+  }
+
+  /**
+   * Waits until {@link WebElement} which defined by {@code element} does not contain the specified
+   * {@code expectedText}.
+   *
+   * <p>Note! Text is extracted by {@link WebElement#getText()} method.
```

Comment:
Also, you should create an override for a case where text extracted by "getAttribute"